lib/service/logger.js

```diff
@@ -18,15 +18,17 @@ function _log(level, message) {
   if (isNil(message)) return;
   if (message instanceof Error) message = message.toString();
 
-  const logMessage = JSON.stringify({
+  const logMessage = {
     message,
     level,
     dlcId: process.env.DLC_ID,
     backingServerId: process.env.BACKING_SERVER_ID
-  });
+  };
 
-  process.stdout.write(logMessage);
-  return logMessage;
+  const logMessageString = `${JSON.stringify(logMessage)}\n`;
```

Comment:
This is more clearly/cleanly written as `JSON.stringify(logMessage) + '\n'` -- i.e. just concatenating two strings rather than creating a string that embeds another string and adds a single character.